packages/constants/src/lib/constants/constants.ts

```diff
@@ -466,22 +466,50 @@ export const LIT_CHAINS: LITChain<LITEVMChain> = {
     type: null,
     vmType: 'EVM',
   },
+
+  /**
+   * Chainlist entry for the Chronicle Testnet.
+   * https://chainlist.org/chain/175177
+   */
   chronicleTestnet: {
     contractAddress: null,
     chainId: 175177,
     name: 'Chronicle - Lit Protocol Testnet',
-    symbol: 'testLPX',
+    symbol: 'tstLPX',
```

Comment:
Looking at https://chainlist.org/?chain=2311&search=chronicle&testnets=true , I think this is backwards -- it shows vesuvius as `tstLPX` and our existing Chronicle as `tstLIT` ?

Reply:
Ohh nice catch! Fixed!